package.json

```diff
@@ -131,6 +132,9 @@
   "scripts": {
     "lint": "eslint \"**/*.ts\" --fix",
     "lint:ci": "eslint \"**/*.ts\" --max-warnings 0",
+    "pack-hdf-converters": "run-script-os",
+    "pack-hdf-converters:win32": "pack-hdf-converters.bat",
+    "pack-hdf-converters:darwin:linux": "chmod u+r+x pack-hdf-converters.sh && ./pack-hdf-converters.sh",
```

Comment:
why are you setting permissions in the file?  just upload it to git as an executable and it should retain the appropriate permissions.

https://stackoverflow.com/questions/21691202/how-to-create-file-execute-mode-permissions-in-git-on-windows?answertab=trending#tab-top